src/components/CreateCard/CreateCard.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { ChangeEventHandler, useState } from 'react';
+import s from './CreateCard.module.css';
+import { SubmitHandler, useForm } from 'react-hook-form';
+
+import Input from './InputField/Input';
+
+type FormValues = {
+  tripName: string;
+  tripDate: string;
+  tripType: string;
+  overnightStay: string;
+  tripImg: FileList;
+};
+
+const CreateCard = (props: any) => {
```

Comment:
dont't use any, if you don't necessary need props you can use optional field like
type FormValues = {
  tripName?: string;
  tripDate: string;
  tripType?: string;
  overnightStay?: string;
  tripImg: FileList;
};

pay attention on "?" sign this means that field is optional